exercises/practice/palindrome-products/.meta/example.rb

```diff
@@ -1,43 +1,40 @@
 Palindrome = Struct.new(:value, :factors)
 
 class Palindromes
-  attr_reader :range
-  def initialize(options)
-    max = options.fetch(:max_factor)
-    min = options.fetch(:min_factor) { 1 }
-    @range = (min..max)
-  end
+  private
+  attr_reader :min, :max
```

Comment:
```suggestion
  private
  
  attr_reader :min, :max
```

Reply:
vscode kept triggering rubocop and auto-formatting my code. I assumed that that's exercism's style.

I think this is the rule that's being followed:
https://github.com/exercism/ruby/blob/fa35ccc41372394db18b4c4a140e27b2597d38d6/.rubocop.yml#L38